man/systemd-analyze.xml

```diff
@@ -792,6 +792,314 @@ Service b@0.service not loaded, b.socket cannot be started.
+                <entry>RestrictNamespaces_CLONE_NEWUTS</entry>
+              </row>
+              <row>
+                <entry>RestrictNamespaces_CLONE_NEWNET</entry>
+              </row>
+              <row>
+                <entry>RestrictAddressFamilies_AF_INET_INET6</entry>
+              </row>
+              <row>
+                <entry>RestrictAddressFamilies_AF_UNIX</entry>
+              </row>
+              <row>
+                <entry>RestrictAddressFamilies_AF_NETLINK</entry>
+              </row>
+              <row>
+                <entry>RestrictAddressFamilies_AF_PACKET</entry>
+              </row>
+              <row>
+                <entry>RestrictAddressFamilies_OTHER</entry>
+              </row>
+              <row>
+                <entry>SystemCallArchitectures</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_swap</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_obsolete</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_clock</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_cpu_emulation</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_debug</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_mount</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_module</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_raw_io</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_reboot</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_privileged</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_resources</entry>
+              </row>
+              <row>
+                <entry>IPAddressDeny</entry>
+              </row>
+              <row>
+                <entry>DeviceAllow</entry>
+              </row>
+              <row>
+                <entry>AmbientCapabilities</entry>
+              </row>
+            </tbody>
+          </tgroup>
+      </table>
+
+      <example>
+        <title>JSON Policy</title>
+        <para>The JSON file passed as a path parameter to <option>--security-policy=</option>
+        must be formatted correctly. The keys should be strings consisting of only alphanumeric
+        characters and underscores. The values should be JSON objects with one or more of the
+        following fields: description_na (string), description_good (string), description_bad
+        (string), weight (unsigned integer), and range (unsigned integer). If any of these fields
+        corresponding to a specific id of the unit file is missing from the JSON object, the
+        hard-coded field value corresponding to that same id is used for security analysis
```

Comment:
I'd say "default built-in" instead of "hard-coded". The latter sounds so needlessly negative.